Work around linker issues by manually defining the symbols

Not suitable for upstream

--- a/devhelp/symbol.map	2021-09-22 15:12:22.721966350 -0700
+++ b/devhelp/symbol.map	2021-09-22 16:26:36.195889506 -0700
@@ -1,6 +1,68 @@
 {
 global:
-  dh_*;
+  dh_genkey_pair;
+  dh_key_derive;
+  dh_book_get_title;                   
+  dh_finalize;                         
+  dh_sidebar_set_search_string;        
+  dh_book_list_get_books;              
+  dh_web_view_can_zoom_out;            
+  dh_search_bar_new;                   
+  dh_notebook_new;                     
+  dh_book_list_builder_new;            
+  dh_settings_is_book_enabled;    
+  dh_tab_get_web_view;                 
+  dh_settings_get_group_books_by_language;
+  dh_assistant_view_get_type;          
+  dh_book_get_type;                   
+  dh_settings_thaw_books_disabled_changed;
+  dh_sidebar_set_search_focus;         
+  dh_search_bar_grab_focus_to_search_entry ;
+  dh_book_cmp_by_title;               
+  dh_application_window_bind_sidebar_and_notebook; 
+  dh_book_list_builder_add_default_sub_book_lists; 
+  dh_web_view_zoom_out;               
+  dh_web_view_get_devhelp_title;     
+  dh_web_view_can_zoom_in;            
+  dh_book_list_builder_create_object;  
+  dh_settings_freeze_books_disabled_changed;
+  dh_web_view_can_reset_zoom;        
+  dh_tab_get_type;                     
+  dh_settings_set_book_enabled;       
+  dh_init;                            
+  dh_link_unref;                       
+  dh_assistant_view_search;           
+  dh_notebook_get_active_web_view;    
+  dh_web_view_reset_zoom;              
+  dh_web_view_zoom_in;                 
+  dh_sidebar_get_selected_link;        
+  dh_book_get_language;               
+  dh_notebook_open_new_tab;            
+  dh_sidebar_new2;                     
+  dh_settings_get_default;             
+  dh_link_get_name;                    
+  dh_book_manager_get_type;            
+  dh_search_bar_get_type;              
+  dh_keyword_model_get_type;           
+  dh_book_tree_get_type;               
+  dh_link_type_get_type;              
+  dh_book_list_builder_get_type;       
+  dh_book_list_get_type;               
+  dh_assistant_view_get_type;         
+  dh_book_get_type;                    
+  dh_settings_builder_get_type;        
+  dh_book_list_directory_get_type;    
+  dh_tab_get_type;                     
+  dh_profile_get_type;                 
+  dh_profile_builder_get_type;         
+  dh_notebook_get_type;                
+  dh_completion_get_type;              
+  dh_tab_label_get_type;               
+  dh_link_flags_get_type;              
+  dh_sidebar_get_type;                
+  dh_web_view_get_type;                
+  dh_settings_get_type;               
+  dh_link_get_type;                    
 local:
   *;
 };
